spec/factories/datacite_endpoint.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
```

Comment:
Can I just say, I'm a massive fan of people who leave a space below the `#frozen_string_literal`. ❤️